### ISSUE TITLE
AttributeError: 'kivy.graphics.svg.Svg' object has no attribute '_source'

### ISSUE THREAD

**Issue by HeyITGuyFixIt** [kivy/kivy#6699]
I am trying out version 2.0.0rc1 of Kivy and Python 3.8.1 on Windows 10. I also tried experimenting with SVGs as I would like to use them in my program. So I was testing out the example scripts and got this error:
```
 Traceback (most recent call last):
   File "main.py", line 50, in <module>
     SvgApp().run()
   File "C:\Users\HP Owner\AppData\Local\Programs\Python\Python38-32\lib\site-packages\kivy\app.py", line 949, in run        self._run_prepare()
   File "C:\Users\HP Owner\AppData\Local\Programs\Python\Python38-32\lib\site-packages\kivy\app.py", line 919, in _run_prepare
     root = self.build()
   File "main.py", line 43, in build
     svg = SvgWidget(filename, size_hint=(None, None))
   File "main.py", line 29, in __init__
     svg = Svg(filename)
   File "kivy\graphics\svg.pyx", line 422, in kivy.graphics.svg.Svg.__init__
 AttributeError: 'kivy.graphics.svg.Svg' object has no attribute '_source'
```
This refers to line 422 of this file:
https://github.com/kivy/kivy/blob/ae0fa0cab3025e517452f44b91dd7d95182f39b2/kivy/graphics/svg.pyx#L422
This error was thrown in line 29 of the SVG example's main.py file:
https://github.com/kivy/kivy/blob/a4505ae6afbb4f6941ce491478e54fe051b11efd/examples/svg/main.py#L29

I can't see why this error occurred. Any ideas why this happened?

**Comment by HeyITGuyFixIt** [comment]
May be related in some way to #6110

**Comment by Commander-Firestrike** [comment]
I believe it is related to #6467, since this mentions changing "filename" to "source"

**Comment by HeyITGuyFixIt** [comment]
> I believe it is related to #6467, since this mentions changing "filename" to "source"

You are probably right and I think this is more related to that than what I thought. But I still don't understand why I got this error. The error suggests the _source property was never created. However, looking at where the error is coming from, the _source property is being created at line 422 where the error is coming from.

My knowledge of Python isn't that extensive, and is a bit rusty right now. So maybe I am missing why this error is being thrown.

**Comment by ilfirin-ms** [comment]
Not in cython, definition in svg.pxd stayed as 60: cdef str _filename

**Comment by pythonic64** [comment]
Solved in `2.0.0rc2` and newer versions.
